### ISSUE TITLE
Remapped URLs should appear only once

### ISSUE THREAD

**Issue by warpech** [StarcounterApps/Website#14]
After PR #13 is merged, I would like to discuss the following problem. When a partial is mapped both using `UriMapping.OntologyMap` and the Website `WebMap`, it can be called twice - therefore appear twice in the merged output. 

Right now I am just making the issue for future reference.

### Steps to reproduce

1. Start the sample config following the steps in https://github.com/StarcounterPrefabs/Website/blob/compatibility-fix/README.md
2. Go to http://localhost:8080/apps

### Expected result

SignIn form and Registration form should appear once

### Actual result

SignIn form and Registration form should appear twice

##### Involved apps' version

All from `develop`

### Screenshot

![2017-01-19 17_51_24-content](https://cloud.githubusercontent.com/assets/566463/22117035/bcfb8e98-de72-11e6-9613-03a08568eba4.png)

**Comment by warpech** [comment]
@chrhol your help on this one will be needed, too.

**Comment by warpech** [comment]
Ping @chrhol any news on this? This is very important for the Launcher revamp project.

**Comment by warpech** [comment]
I've discussed this with @chrhol and he suggests that it works as desired. It should be fixed by unmapping "Registration" form from "Sign In" form. 

If we want to show them together, we can map both explicitly it an "Unauthorized page" in any app.

@miyconst do you agree?

**Comment by warpech** [comment]
Assigning this to me to apply the fix proposed by @chrhol

**Comment by warpech** [comment]
As discussed earlier in this thread, this issue is really out of scope of WebsiteEditor/WebsiteProvider. It depends on the usage of `Blender.MapUri` within apps.

IMO there should be a way to disable maps created elsewhere (perhaps using `Blender.UnmapUri`). Should this be a feature of WebsiteEditor/WebsiteProvider or Starcounter Administrator? Let's discuss.

cc @alemoi

**Comment by un-tone** [comment]
I guess with the currrent WebsiteProvider implementation and last changes in the platform (https://github.com/StarcounterApps/Website/issues/128), the problem is gone.

**Comment by warpech** [comment]
I agree. I have moved the proposal from my last comment to a separate issue: https://github.com/Starcounter/RebelsLounge/issues/147
